.github/workflows/deploy-with-docker.yml

```diff
@@ -39,7 +39,7 @@ jobs:
           docker logs docker-deploy_managerapi_1
 
       - name: Run Test
-        run: api/test/shell/docker_deploy_test.sh
+        run: api/test/shell/docker_deploy_test.sh -s false
```

Comment:
we could change the filename now, since it's not just for docker deploy test.

Reply:
Done

Reply:
Rename to `manager_smoking.sh`